src/handlers/assign.rs

```diff
@@ -7,6 +7,9 @@
 //! * `@rustbot release-assignment`: Removes the commenter's assignment.
 //! * `r? @user`: Assigns to the given user (PRs only).
 //!
+//! Note: this module does not handle review assignments issued from the
```

Comment:
Do you handle this elsewhere?

Reply:
yes, exactly [here](https://github.com/rust-lang/triagebot/pull/1786/files#diff-63f676f3f26949c577c742ff956f9c696a8a640529b4a268068ac205508e4c11R76). 

At this time we don't handle PR assignment done using the GH web UI, so what happens is a bit ugly; in that piece of code I handle the webhook _post-assignment_ and when the assignee(s) don't have capacity I _revert_ the assignment and emit a comment.

(ideally we should handle those assignments in a webhook but I felt this a topic for another pr :slightly_smiling_face: )